apps/backend/src/db/migrations/20240904155242_add_activity_logs_table.ts

```diff
@@ -0,0 +1,61 @@
+import { type Knex } from "knex";
+
+const TABLE_NAME = "activity_logs";
+
+const ColumnName = {
+	COMMITS_NUMBER: "commits_number",
+	CREATED_AT: "created_at",
+	CREATED_BY_USER_ID: "created_by_user_id",
+	DATE: "date",
+	GIT_EMAIL_ID: "git_email_id",
+	ID: "id",
+	PROJECT_ID: "project_id",
+	UPDATED_AT: "updated_at",
+} as const;
+
+const GIT_EMAILS_TABLE = "git_emails";
+const PROJECTS_TABLE = "projects";
+const USERS_TABLE = "users";
+
+function up(knex: Knex): Promise<void> {
+	return knex.schema.createTable(TABLE_NAME, (table) => {
+		table.increments(ColumnName.ID).primary();
+		table.integer(ColumnName.COMMITS_NUMBER).notNullable();
```

Comment:
Given that commits_number should never be negative, i think it would be better to use unsigned() for this column.